recipes-connectivity/openssh/openssh_%.bbappend

```diff
@@ -1,4 +1,16 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+SAVED_DIR := "${THISDIR}"
```

Comment:
Doesn't the `close-all-ssh-connections.path` file get copied into the `WORKDIR` during the `do_unpack`? Or do you need to add it explicitly to our bbappends `SRC_URI`?

Either way bitbake should copy it into the workspace for you, and you shouldn't have to apply it directly from the meta-nilrt source.

Reply:
> Doesn't the `close-all-ssh-connections.path` file get copied into the `WORKDIR` during the `do_unpack`? Or do you need to add it explicitly to our bbappends `SRC_URI`?

Right, unless you specifically add it to the SRC_URI variable, the file is essentially "not in the recipe", so it doesn't get copied to the WORKDIR.

Reply:
Why can't you add the `.patch` to the `SRC_URI` in the bbappend?

Reply:
Because --- when I try, this happens ....


ERROR: openssh-8.5p1-r0 do_patch: Command Error: 'quilt --quiltrc /ssd/hardknott/nilrt/build/tmp-glibc/work/core2-64-nilrt-linux/openssh/8.5p1-r0/recipe-sysroot-native/etc/quiltrc push' exited with 0  Output:
stdout: Applying patch close-all-ssh-connections.patch
can't find file to patch at input line 3
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|--- a/init
|+++ b/init
--------------------------
No file to patch.  Skipping patch.
1 out of 1 hunk ignored
Patch close-all-ssh-connections.patch does not apply (enforce with -f)

stderr: 
ERROR: Logfile of failure stored in: /ssd/hardknott/nilrt/build/tmp-glibc/work/core2-64-nilrt-linux/openssh/8.5p1-r0/temp/log.do_patch.584
ERROR: Task (/ssd/hardknott/nilrt/sources/openembedded-core/meta/recipes-connectivity/openssh/openssh_8.5p1.bb:do_patch) failed with exit code '1'


I suppose I could try to rename the file to something that doesn't end in .patch that do_patch will ignore, but then I might start getting errors about having a file in SRC_URI that wasn't "installed" anywhere.  If you want, I can try that after the Dr. Appointment today.

Reply:
Ok that last idea worked.  I think you've got everything you asked for in this version!  -- And no that wasn't a challenge to find more things wrong.